nwb_healthstatus/producers/spikeextractors/recording.py

```diff
@@ -0,0 +1,226 @@
+            # Nixio - RuntimeError: Cannot open non-existent file in ReadOnly mode!
+            # (
+            #     se.NIXIORecordingExtractor,
+            #     "nix",
+            #     dict(file_path=str(Path("neoraw.nix"))),
+            # ),
+            (
+                se.OpenEphysRecordingExtractor,
+                "openephys/OpenEphys_SampleData_1",
+                dict(folder_path=Path("openephys", "OpenEphys_SampleData_1")),
+            ),
+            (
+                se.OpenEphysRecordingExtractor,
+                "openephysbinary/v0.4.4.1_with_video_tracking",
+                dict(folder_path=Path("openephysbinary", "v0.4.4.1_with_video_tracking")),
+            ),
+            (
+                se.OpenEphysNPIXRecordingExtractor,
+                "openephysbinary/v0.5.3_two_neuropixels_stream",
+                dict(
+                    folder_path=Path("openephysbinary", "v0.5.3_two_neuropixels_stream", "Record_Node_107")
+                ),
+            ),
+            (
+                se.NeuropixelsDatRecordingExtractor,
+                "openephysbinary/v0.5.3_two_neuropixels_stream",
+                dict(
+                    file_path=Path("openephysbinary", "v0.5.3_two_neuropixels_stream", "Record_Node_107", "experiment1", "recording1", "continuous",
+                              "Neuropix-PXI-116.0", "continuous.dat"),
+                    settings_file=Path("openephysbinary",
+                                  "v0.5.3_two_neuropixels_stream", "Record_Node_107", "settings.xml")
+                ),
+            ),
+            (
+                se.PhyRecordingExtractor,
+                "phy/phy_example_0",
+                dict(folder_path=Path("phy" / "phy_example_0")),
+            ),
+            # Plexon - AssertionError: This file have several channel groups spikeextractors support only one groups
+            # (
+            #     se.PlexonRecordingExtractor,
+            #     "plexon",
+            #     dict(filename=Path("plexon", "File_plexon_2.plx")),
+            # ),
+            (
+                se.SpikeGLXRecordingExtractor,
+                "spikeglx/Noise4Sam_g0",
+                dict(
+                    file_path=Path("spikeglx", "Noise4Sam_g0", "Noise4Sam_g0_imec0",
+                    "Noise4Sam_g0_t0.imec0.ap.bin")
+                )
+            )
+        ]
+
+    def create(self):
+        for se_class, dataset_path, se_kwargs in self.SCENARIOS:
+            self.dataset.get(dataset_path)
+            for k in ["filename", "file_path", "dirname", "folder_path", "settings_file"]:
+                if k in se_kwargs:
+                    se_kwargs[k] = self.pt / se_kwargs[k]
+            if "filename" in se_kwargs:
+                se_kwargs["filename"] = str(se_kwargs["filename"])
+            recording = se_class(**se_kwargs)
+            nwbfile = NWBFile()
+            se.NwbRecordingExtractor.write_recording(recording, nwbfile=nwbfile, write_scaled=True)
```

Comment:
@yarikoptic This bit doesn't seem to work; `nwbfile = NWBFile()` seems to be erroring out with `TypeError: NWBFile.__init__: missing argument 'session_description', missing argument 'identifier', missing argument 'session_start_time'`.

Reply:
just add some dummy ones.  This is just for a test after all